src/common/services/session/sessionModal.service.js

```diff
@@ -35,6 +37,7 @@ function SessionModalService( $uibModal, $log, modalStateService, analyticsFacto
       }
     }, options );
     currentModal = $uibModal.open( modalOptions );
+    currentModal.type = type;
```

Comment:
Maybe I shouldn't store this on the uibModal object. I suppose I could add another variable.